.env

```diff
@@ -27,7 +27,9 @@ CAS_SERVER=https://auth.biodiversitydata.se
 CAS_AFTER_LOGIN=main_bp.index
 
 # File upload
-MAX_CONTENT_LENGTH=838860800
+# Size limit is set in nginx (molecular.conf: client_max_body_size 800m;),
+# but also here, to show flask (rather than nginx) error page
```

Comment:
Technically, we need to tell nginx to allow larger uploads so it passes them through.